frontend/src/hooks/usePlants.ts

```diff
@@ -0,0 +1,24 @@
+import {useEffect, useState} from "react";
+import axios from "axios";
+import {Plant} from "../components/Plant";
+
+
+export default function usePlants() {
+
+    const [plants, setPlants] = useState<Plant[]>([])
```

Comment:
Bonus: vielleicht ist es noch schöner keinen Initialwert für useState anzugeben.

```suggestion
    const [plants, setPlants] = useState<Plant[]>()
```